deploy-cloudgov-docker.sh

```diff
@@ -35,51 +49,62 @@ echo DOCKER_BACKEND_IMAGE: $DOCKER_IMAGE_BACKEND
 echo DOCKER_FRONTEND_IMAGE: $DOCKER_IMAGE_FRONTEND
 echo CIRCLE_BRANCH=$CIRCLE_BRANCH
 
+# EXAMPLES
+
+# For an example rolling deploy:
+# See .circleci/config.yml, which calls this script as part of the deploy process.
+
+# Example vendor staging deploy:
+# ./deploy-cloudgov-docker.sh setup test tdp-backend-vendor-staging tdp-frontend-vendor-staging lfrohlich/tdp-backend:raft-tdp-main lfrohlich/tdp-frontend:raft-tdp-main raft-tdp-main
+
+# Remember, the script assumes that the user is logged in to Cloud.gov via the Cloud Foundry CLI.
+
+# SCRIPT HELPERS AND TASKS
 
-# function to check if a service exists
+# Helper function to check if a service exists.
 service_exists()
 {
   cf service "$1" >/dev/null 2>&1
 }
 
 
-# Performs a normal deployment unless rolling is specified in the fucntion call
+# Performs a normal deployment unless rolling is specified:
 update_frontend()
 {
 	if [ "$1" = "rolling" ] ; then
-		# Do a zero downtime deploy.  This requires enough memory for
-		# two apps to exist in the org/space at one time.
-		#The `--var` parameter ingest a value into the ((docker-frontend)) environment variable in the manifest.yml**
+		# Do a zero downtime deploy.
+		# This requires enough memory for two apps to exist in the org/space at one time.
+		# The `--var` parameter ingests a value into the ((docker-backend)) environment variable in the manifest.yml.
 		cf push $CGHOSTNAME_FRONTEND --no-route -f tdrs-frontend/manifest.yml --var docker-frontend=$DOCKER_IMAGE_FRONTEND --strategy rolling || exit 1
 	else
 		cf push $CGHOSTNAME_FRONTEND --no-route -f tdrs-frontend/manifest.yml --var docker-frontend=$DOCKER_IMAGE_FRONTEND
 	fi
 	cf map-route $CGHOSTNAME_FRONTEND app.cloud.gov --hostname "${CGHOSTNAME_FRONTEND}"
 }
 
-# Performs a normal deployment unless rolling is specified in the fucntion call
+# Performs a normal deployment unless rolling is specified:
 update_backend()
 {
 	if [ "$1" = "rolling" ] ; then
-		# Do a zero downtime deploy.  This requires enough memory for
-		# two apps to exist in the org/space at one time.
-		#The `--var` parameter ingest a value into the ((docker-backend)) environment variable in the manifest.yml**
+		# Do a zero downtime deploy.
+		# This requires enough memory for two apps to exist in the org/space at one time.
+		# The `--var` parameter ingests a value into the ((docker-backend)) environment variable in the manifest.yml.
 		cf push $CGHOSTNAME_BACKEND --no-route -f tdrs-backend/manifest.yml --var docker-backend=$DOCKER_IMAGE_BACKEND --strategy rolling || exit 1
 
 	else
 		cf push $CGHOSTNAME_BACKEND --no-route -f tdrs-backend/manifest.yml --var docker-backend=$DOCKER_IMAGE_BACKEND
-		# set up JWT key if needed
+		# Set up JWT key if needed:
 		if cf e $CGHOSTNAME_BACKEND | grep -q JWT_KEY ; then
-		   echo jwt cert already created
+			echo "jwt cert already created."
 		else
-		   generate_jwt_cert
-	   fi
+			echo "jwt cert needs to be created and set: see generate_jwt_cert."
+		fi
```

Comment:
This should be the only non-code-comment change introduced in this PR; described in more detail in the PR body.